src/display/editor/editor.js

```diff
@@ -785,6 +805,38 @@ class AnnotationEditor {
     this.fixAndSetPosition();
   }
 
+  addAltTextButton() {
+    if (this.#altTextButton) {
+      return;
+    }
+    const altText = (this.#altTextButton = document.createElement("span"));
+    altText.className = "altText";
+    altText.dir = this._uiManager.direction;
```

Comment:
This is probably a stupid question on my part, but further down you're setting the `dir`-property on the *entire* `annotationEditorLayer`, so shouldn't the `altText` DOM-elements inherit that without us having to specify it explicitly here?